packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -197,4 +202,23 @@ const IModelLoader = React.memo(
   }
 );
 
+/**
+ * Create a blank connection with default props
+ * @param blankConnectionProps?
+ * @returns BlankConnection
+ */
+const createBlankConnection = (
```

Comment:
if we're going to go with the approach where you only need an iTwinId then we should pass it as another arg here,
https://www.itwinjs.org/reference/core-frontend/imodelconnection/blankconnectionprops/

Reply:
Yes, makes sense. Good catch.

Reply:
idk if the arg should be required, because what if the user provides blankConnection instead?
i guess it depends on what we do with that prop, and if we can remove it w/o a breaking change

Reply:
Was working under the assumption that iTwinId was always required, but I see now.

Reply:
I can just make the blankConnection required again, if that is easiest

Reply:
Reworked.